### DISCUSSION TITLE
Not all the Lang will be published.

### DISCUSSION THREAD

**Discussion by mjamilasfihani** [milwad-dev/laravel-validate#30] [General]
Hi! I've found some potential bug in the future, as you can see I've took some minutes to compare between the lang dirs and the attribute and not all of them was added into `$langs`.

While for publishing from `publishLangFiles()` method you were loop these values first before execute the publisher. Maybe we can add new tests to make sure everything was published correctly? Thank you.

![image](https://user-images.githubusercontent.com/51300528/232854403-58169ecb-c3ff-44ba-bbe3-674114ab085c.png)

**Comment by milwad-dev** [comment]
Yes, I saw it in your PR https://github.com/milwad-dev/laravel-validate/pull/29.
I add seperate PR for this, and added test.

Thanks!
